packages/cmf/src/http.js

```diff
@@ -0,0 +1,274 @@
+import merge from 'lodash/merge';
+import get from 'lodash/get';
+import { mergeCSRFToken } from './middlewares/http/csrfHandling';
+import { HTTP_STATUS, testHTTPCode, HTTP_METHODS } from './middlewares/http/constants';
+
+/**
+ * Storage point for the doc setup using `setDefaultConfig`
+ */
+export const HTTP = {
```

Comment:
Can you not export that please ?
There is a getter, that is part of the api. Let's keep it internal.

Reply:
Is it used in the tests ? (but even if, it's not a good reason :D)

Reply:
it seems to be used in the tests to reset its state. You can expose a reset of the tests can get + mutate